uwblueprint/supportive-housing
# Edit and Delete Resident

## GitHub Issue link

[Issue Name](https://github.com/uwblueprint/supportive-housing/issues/)
#92 


## Implementation description
* use API to edit and delete employees
* created modal to allow for employee editing and deletion


## Steps to test
1. Add multiple users to the resident database
2. Test editing basic features like initials, move in date, etc
3. Test check boxing and unboxing the resident move out date and seeing if it is able to be edit
4. Test invalid and valid move in/move out date combinations
5. Test deletion


## What should reviewers focus on?
* 


## Checklist
